### PR TITLE
Remove raw and template

### LINKED ISSUE #386: Alert.yaml file should be plenty data only
https://github.com/SUSE/ha-sap-terraform-deployments/commit/07952934aad8f405d40890c00ad4e9cf7d110108

we should remove `raw` so user can copy and paste directly

### PR DESCRIPTION
**Pull request by MalloZup**
fix #386

### CODE REVIEW

**Comment by diegoakechi** [comment]
@MalloZup Did you test this? I think it will not work. The https://github.com/SUSE/ha-sap-terraform-deployments/blob/master/salt/monitoring/prometheus/prometheus.yml has Jinja interpolations on it, so I think this change will copy the file as it is and the prometheus server will not start. Or maybe I am missing something here...

**Comment by MalloZup** [comment]
the change works itself. the `prometheus.yaml` file we cannot avoid jinja there.

I will do a more refactor for making it clear

File: salt/monitoring/init.sls [hunk 1]
```diff
@@ -20,22 +20,31 @@ prometheus:
...
-    - name: /etc/prometheus/
-    - makedirs: True
-    - source: salt://monitoring/prometheus
+  file.managed:
+    - name:  /etc/prometheus/prometheus.yml
+    - source: salt://monitoring/prometheus/prometheus.yml.j2
     - template: jinja
-    - include_empty: True
+    - require:
+      - pkg: prometheus
```
Comment (stefanotorresi) [inline salt/monitoring/init.sls:36]: Can we get rid of the `shap` prefix from the name of these two states? We don't really need that and external contributors won't know what that means.

Comment (MalloZup) [reply salt/monitoring/init.sls:36]: make sense thx!

**Pull request merged**